src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -1367,6 +1435,87 @@ RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1,
 #endif // backend
 }
 
+/******************** jitter ********************/
+
+RppStatus rppt_jitter_gpu(RppPtr_t srcPtr,
+                          RpptDescPtr srcDescPtr,
+                          RppPtr_t dstPtr,
+                          RpptDescPtr dstDescPtr,
+                          Rpp32u *kernelSizeTensor,
+                          Rpp32u seed,
+                          RpptROIPtr roiTensorPtrSrc,
+                          RpptRoiType roiType,
+                          rppHandle_t rppHandle)
+{
+#ifdef HIP_COMPILE
+    Rpp32u paramIndex = 0;
+    copy_param_uint(kernelSizeTensor, rpp::deref(rppHandle), paramIndex++);
```

Comment:
Please remove this copy in HIP API
since in future, we want to accept all the params either in pinned/HIP memory to avoid this copies